File: allogcrash_release/src/main/java/com/alguojian/crash/CrashHandler.java
package com.alguojian.crash;

import android.content.Context;

public class CrashHandler {

    private static CrashHandler INSTANCE = new CrashHandler();

    private CrashHandler() {
    }

    /**
     * 获取CrashHandler实例 ,单例模式
     */
    public static CrashHandler getInstance() {
        return INSTANCE;
    }


    public void init(Context context) {
    }


    public void setOtherNews(Object otherNews) {
    }


    public void setDingDingLink(String s) {
    }

}

File: allogcrash_release/src/main/java/com/alguojian/logcrash/CrashHandler.kt
package com.alguojian.logcrash

import android.app.Application

object CrashHandler {
    @JvmStatic
    @JvmOverloads
    fun initThis(context: Application, alreadyUsed: Boolean = false) {
    }

    @JvmStatic
    fun setOtherNews(hashMap: HashMap<String, String?>) {

    }

    @JvmStatic
    fun setDingDingLink(s: String?) {
    }

    @JvmStatic
    fun postCrashToDingding() {
    }
}